tests/test_intermediary.py

```diff
@@ -3,199 +3,204 @@
 
 import sync2jira.intermediary as i
 
-PATH = 'sync2jira.intermediary.'
+PATH = "sync2jira.intermediary."
 
 
 class TestIntermediary(unittest.TestCase):
     """
     This class tests the downstream_issue.py file under sync2jira
     """
+
     def setUp(self):
         self.mock_config = {
-            'sync2jira': {
-                'map': {
-                    'github': {
-                        'github': {'mock_downstream': 'mock_key'}
-                    }
-                }
+            "sync2jira": {
+                "map": {"github": {"github": {"mock_downstream": "mock_key"}}}
             }
         }
```

Comment:
Here's a place where I would have used Black's "magic comma" to preserve the old formatting:
```suggestion
        self.mock_config = {
            'sync2jira': {
                'map': {
                    'github': {
                        'github': {'mock_downstream': 'mock_key'}
                    },
                },
            },
        }
```

Reply:
I'll let this one go. The reason I love black is that I get to stop thinking about things like this and focus on other things :)